kieserver/image.yaml

```diff
@@ -142,6 +142,9 @@ envs:
   - name: "KIE_SERVER_DISABLE_KC_VERIFICATION"
     example: "true"
     description: "Disable Maven KIE Jar verification. It is recommended to test the kjar manually before disabling this verification."
+  - name: "KIE_SERVER_JBPM_CLUSTER"
+    example: "true"
+    description: "Enable the JPM clustering"
```

Comment:
please elaborate a little more the description and add its default value as well.